src/allmydata/scripts/create_node.py

```diff
@@ -344,6 +350,54 @@ def create_node(config):
         os.mkdir(basedir)
     write_tac(basedir, "client")
 
+    # if we're doing magic-wormhole stuff, do it now
+    if config['join'] is not None:
+        print >>out, "Opening wormhole with code '{}'".format(config['join'])
+
+        wh = wormhole.create(
+            appid=u"tahoe-lafs.org/lafs",
+            relay_url=u"ws://wormhole.leastauthority.com:4000/v1",
```

Comment:
I suppose this is still meant to change - but not until wormhole.tahoe-lafs.org gets configured?  Though I don't see the harm in merging this as-is if setting up that alternate name takes too long.  It would be a simple matter to update it in the future after DNS configuration is done.